types/ApiResponses.ts

```diff
@@ -301,6 +301,7 @@ export interface PublicProfile {
 export interface PrivateProfile extends PublicProfile {
   email: string,
   accessType: string,
+  resumeOptedIn: boolean,
   state: string,
   credits: number,
```

Comment:
Should we add this field to `PublicProfile` instead? `PublicProfile` is returned when a user views another user's profile, so depending on the behavior of "opting-in", we may or may not need to include this field in the public response, so that the frontend knows to display/not display the resume when another user views a profile

Reply:
We don't actually want public users, in any case, to see whether an individual user is allowing the recruiters to see their resume as that is not necessary.